config/initializers/omniauth.rb

```diff
@@ -1,14 +1,18 @@
 require 'moj_sso'
 
-# Localhost SSO config
-app_id = ENV.fetch('MOJSSO_ID')
-app_secret = ENV.fetch('MOJSSO_SECRET')
-sso_url = ENV.fetch('MOJSSO_URL', 'http://localhost:5000') # <- default for localhost
+Rails.configuration.auth_enabled = ENV.fetch('AUTH_ENABLED', 'true').downcase != 'false'
```

Comment:
`auth_enabled` returns a boolean so the Ruby convention is to name it with a question mark, `auth_enabled?`